server/graphql/v2/object/Host.js

```diff
@@ -427,6 +442,113 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          accounts: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          from: {
+            type: GraphQLString,
+            description: "Inferior date limit in which we're calculating the contribution stats",
+          },
+          to: {
+            type: GraphQLString,
+            description: "Superior date limit in which we're calculating the contribution stats",
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the contribution stats.');
+          }
+          const where = { HostCollectiveId: host.id, kind: 'CONTRIBUTION' };
+          const dateRange = dateRangeHelper(args.from, args.to);
+          if (dateRange) {
+            where.createdAt = dateRange;
+          }
+          if (args.accounts) {
+            const collectiveIds = args.accounts.map(account => account.legacyId);
+            where.CollectiveId = { [Op.in]: collectiveIds };
+          }
+          const contributions = await models.Transaction.findAndCountAll({
```

Comment:
:no_entry_sign: Here we're loading all transactions (without `SELECT`ing the fields) in RAM to sum them in JS. If we take OSC as a reference, without filtering by collectives (default), that's ~460,000 transactions. This will probably crash or timeout. Let's COUNT and SUM in SQL, even if that's mean having two queries.